tools/ci_build/github/azure-pipelines/templates/windows-code-coverage-steps.yml

```diff
@@ -1,20 +1,35 @@
-# sets variables $(TestDataUrl) and $(TestDataChecksum)
+# Runs steps to generate and publish code coverage data
 
 parameters:
   OpenCppCoverageExe: '$(Build.BinariesDirectory)\OpenCppCoverage\OpenCppCoverage.exe'
-
+  GitCommitHash: $(OnnxRuntimeGitCommitHash)
+  PublishReport: true
+  PostToDashBoard: false
+  ContinueOnError: false
 steps:
 
 - task: PowerShell@2
   displayName: 'Run Test Coverage'
   inputs:
     filePath: '$(Build.SourcesDirectory)\tools\ci_build\github\windows\run_OpenCppCoverage.ps1'
     arguments: '-OpenCppCoverageExe:"${{parameters.OpenCppCoverageExe}}" -SourceRoot:"$(Build.SourcesDirectory)" -BuildRoot:"$(Build.BinariesDirectory)" -LocalBuild:$false'
-  continueOnError: true  
+  continueOnError: ${{parameters.ContinueOnError}}  
 
 - task: PublishCodeCoverageResults@1
   inputs:
     codeCoverageTool: 'Cobertura'
     summaryFileLocation: '$(Build.BinariesDirectory)/**/cobertura.xml'
     reportDirectory: '$(Build.BinariesDirectory)/**/OpenCppCoverageResults'
-  continueOnError: true
+  continueOnError: ${{parameters.ContinueOnError}}
```

Comment:
why use parameters (i.e. add configuration complexity)? When would this be set to 'false'?

Reply:
originally defined it to allow CI builds to take CC as optional, and nightly build to take it mandator step. Would leave as is, in case we want to add to CI later.